Automattic/wp-calypso
# Checkout: fix chat icon margin

This adds some margin to the pre-sales Checkout HappyChat button. It also removes the old unused component (from legacy Checkout) and rewrites the current one to use TypeScript.

**Before:** | **After:**
------------ | -------------
<img width="342" alt="Screen Shot 2021-01-06 at 12 59 20 PM" src="https://user-images.githubusercontent.com/942359/103804006-685bff80-501f-11eb-99f9-4cee4291e7ca.png"> | <img width="348" alt="Screen Shot 2021-01-06 at 12 58 58 PM" src="https://user-images.githubusercontent.com/942359/103804027-70b43a80-501f-11eb-8e2d-90d1bcf277b5.png">

**To test:**
- force the pre-sales HappyChat button to render by setting [this](https://github.com/Automattic/wp-calypso/blob/fix%2Fcheckout-payment-chat-styles/client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.js#L341) and [these](https://github.com/Automattic/wp-calypso/blob/fix/checkout-payment-chat-styles/client/components/happychat/button.jsx#L110-L111) to `true`
- add a product or plan to your cart to visit Checkout
- verify that the button matches the screenshots above
- verify that there are no instances in the codebase linking to the old `components/blocks/` component
